doc/htmldoc/_ext/VersionSyncRole.py

```diff
@@ -54,7 +53,7 @@ def role(name, rawtext, text, lineno, inliner, options={}, content=[]):
         else:
             level = "min"
 
-        with open(str(doc_build_dir) + '/_ext/versions.json') as fp:
+        with open(Path(__file__).parent.join('versions.json')) as fp:
```

Comment:
Wouldn't this also work and be more readable?
```suggestion
        with open(Path(__file__).parent / 'versions.json') as fp:
```